Dockerfile

```diff
@@ -1,22 +1,8 @@
-# This is a multi-stage Dockerfile and requires >= Docker 17.05
-# https://docs.docker.com/engine/userguide/eng-image/multistage-build/
-FROM gobuffalo/buffalo:v0.12.7 as builder
+FROM gobuffalo/buffalo:v0.12.7
 
 RUN mkdir -p $GOPATH/src/github.com/cpjudge/cpjudge_webserver
 WORKDIR $GOPATH/src/github.com/cpjudge/cpjudge_webserver
 
-ADD . .
-RUN dep ensure
-RUN buffalo build --static -o /bin/app
-
-FROM alpine
```

Comment:
Previously, the app was running in production mode in the docker container, right? Now the file just runs it in dev mode?
Should we comment the build and run binary part and comment it out to use that when required?

Reply:
Nope, the app was not running in production mode. In fact, the production mode variable [`GO_ENV`](https://github.com/cpjudge/cpjudge-webserver/blob/2d933f44d2e7df4e53dab5d647b2dc9a29e251f9/Dockerfile#L7) is there in the Dockerfile and we can uncomment that line to run it in production.

As for including the build and run binary part in the Dockerfile, we can't do that as the code directory is mounted instead of copied to the container, so the code isn't available during the "build" process of the image.